WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -69,13 +69,13 @@ def select_nearby_delegate
 
   def edit_avatar_thumbnail
     @user = user_to_edit
-    return if redirect_if_cannot_edit_user(@user)
+    return if redirect_unless_user(:can_change_users_avatar?, @user)
```

Comment:
Ah, so *that's* what `and` is for! I just knew to avoid it it =)

Renaming `redirect_unless_user` to `redirect_to_root_unless_user` sounds good to me. Wanna send in a PR? =)